SharkCage/SharedFunctionality/tokenLib/tokenManipulation.h

```diff
@@ -10,11 +10,9 @@
 namespace tokenLib {
 	/**
 	* Function gets a SID of a group and creates a token with a group entry added in the token
-	* Source of the token is the token of the process itself. Returned token is identical to the calling process token, just includes one more group
-	* SE_CREATE_TOKEN_NAME must be held and enabled by a calling process, to successfully call this method
-	* Mind that there are security implications to the current implementations - such that f.x.:
-	* every process launched with token created by this function has SE_CREATE_TOKEN_NAME privilege, granting it basically any access to the system
-	* This can be mitigated by calling CreateRestrictedToken() function on the output of this method
+	* Source of the token is token of the user attached to active physical console session. 
```

Comment:
It is not neccessary anymore... `CreateRestrictedToken` was called because the library used to modify the token of calling process and `CageManager` was running under `NT AUTHORITY` and we did not want to grant a caged program with system level privileges. 

However, now caged processes are launched with the token of user attached to physical console (which makes more sense in context of what the shark cage is trying to achieve) and thus no restriction is needed

Reply:
Cool, just wanted to make sure we are not overlooking something :+1: